### PR TITLE
Updating dlls

### LINKED ISSUE #589: MoBi should be able to import level 3 sbml files

### PR DESCRIPTION
**Pull request by abdelr**
WIP #589 (loading level 3 sbml files)

### CODE REVIEW

**Comment by msevestre** [comment]
hum..... Are you sure swapping the dll is enough to FIX the issue? There is a ton load of mapping going on and if new constructs are necessary, new code will be required

**Comment by abdelr** [comment]
This is why it is on draft mode ;). We are testing ....

**Comment by msevestre** [comment]
sorry did not see. The commit message said "Fixes #589" that's why:)

**Comment by Yuri05** [comment]
@abdelr Which libsml version is it now? Seems to be 5.18. 
Also I would add an integration test with SBML level 3 model.

**Comment by abdelr** [comment]
> @abdelr Which libsml version is it now? Seems to be 5.18.

5.18.0 indeed.

> Also I would add an integration test with SBML level 3 model.
>

Sure, I could already add a couple of integration tests with simple level 2 and level 3 already working tests. But I would say it is a work in progress draft PR for now. We are planning to test more thoroughly and create issues as they are detected. It was not possible to lead level 3 files at all. It was crashing. Now, it does not crash but I am not sure how it handles all the information. We will be testing and fixing issues that could come down the road.
 
> And: does it fix #287 as well?

It crashes when loading the level 2 version 4 file. This might be a good point to try to start solving already.

**Comment by abdelr** [comment]
> sorry did not see. The commit message said "Fixes #589" that's why:)

I added the comment to link the issue but it is not solving the issue yet. I actually thought on not creating the PR but I went for creating the PR on draft mode and work on it so there is somewhere on Github in case we need to cooperate.

**Comment by msevestre** [comment]
That's fine. My suggestion would be to say WIP instead of Fixes (I was on my mail client and did not see the draft mode)

**Comment by abdelr** [comment]
I just added handling invalid conversion. Still, many situations are not handled so I would say this is far from finished (the old importer too). I suggest we start testing it with real files and tackle the issues as they are discovered. I tried some online files and that are not supported. I will try at least that it does not crash but still, we need to describe what information is been discarded that we still need.

File: src/MoBi.Engine/Sbml/SBMLTask.cs [hunk 1]
```diff
@@ -104,9 +104,14 @@ public Model GetModel(string filename)
...
             throw new MoBiException(SBMLConstants.ModelNotRead(sbmlDoc.getErrorLog().ToString()));
          }
-
-         convertSBML(sbmlDoc);
-
+         try
+         {
+            convertSBML(sbmlDoc);
+         }
+         catch (Exception _)
```
Comment (msevestre) [inline src/MoBi.Engine/Sbml/SBMLTask.cs:111]: Are we sure this is always the reason for an exception? Cannot we be more explicit here? What if the code in `convertSBML` (our code) crashes?

**Review by msevestre — CHANGES_REQUESTED** [review]
Asking to see if we can not catch all exception but a more specific one

**Comment by abdelr** [comment]
This is off dated... The comment from @msevestre 

> Asking to see if we can not catch all exception but a more specific one

is fixed on #599

**Pull request closed without merging**